mm2src/mm2_main/src/lp_native_dex.rs

```diff
@@ -400,9 +404,9 @@ fn fix_shared_dbdir(ctx: &MmCtx) -> MmInitResult<()> {
 }
 
 #[cfg(not(target_arch = "wasm32"))]
-fn migrate_db(ctx: &MmArc) -> MmInitResult<()> {
-    let migration_num_path = ctx.dbdir().join(".migration");
-    let mut current_migration = match std::fs::read(&migration_num_path) {
+fn migrate_db(ctx: &MmArc, db_id: Option<&str>) -> MmInitResult<()> {
```

Comment:
https://github.com/KomodoPlatform/komodo-defi-framework/blob/9a1f89d82c6421b3b239391c6f482819ba184145/mm2src/mm2_main/src/lp_native_dex.rs#L497

Reply:
Yes but I can't understand what migrate_db fn does:
it reads current_migration from '.migration' file and if current_migration < 1 it runs an empty migration_1() function then writes current_migration of 1 back to '.migration' file.
Looks like kind of a leftover function.

Reply:
migration_1 might be intentionally left empty as a placeholder for future migrations or is indeed a left over. NO issue with migration if disabled By the way,..so I guess it can be safely removed